sdk/js-cloud-server/src/models/variable.ts

```diff
@@ -58,3 +58,17 @@ export class DVCVariable<
         this.type = type
     }
 }
+
+export class VariableMetadata {
+    constructor(public featureId?: string) {}
+}
+
+export class VariableWithMetadata<T extends DVCVariableValue> {
```

Comment:
nit, this should be VariableAndMetadata since they are both properties of the object, instead of metadata added to the variable